src/main/java/core/basesyntax/service/impl/CsvFileDataHandlerImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.dao.FruitsDaoImpl;
+import core.basesyntax.operations.OperationStrategy;
+import core.basesyntax.service.CsvFileDataHandler;
+import java.util.List;
+
+public class CsvFileDataHandlerImpl implements CsvFileDataHandler {
+    private static final int FIRST_DATA = 1;
+    private static final int OPERATION_INDEX = 0;
+    private static final int PRODUCT_NAME_INDEX = 1;
+    private static final int PRODUCT_AMOUNT_INDEX = 2;
+    private final FruitsDao fruitsDao;
+    private final OperationStrategy operationStrategy;
+
+    public CsvFileDataHandlerImpl(OperationStrategy operationStrategy) {
+        fruitsDao = new FruitsDaoImpl();
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public FruitsDao processData(List<String> fileData) {
```

Comment:
```suggestion
    public void processData(List<String> fileData) {
```

I think this method should be void, it is quite unexpected that it returnsFruitsDao instance